tests/e2e/backup_restore_suite_test.go

```diff
@@ -321,5 +331,14 @@ var _ = Describe("AWS backup restore tests", func() {
 			PreBackupVerify:      mysqlReady(true, false, CSIDataMover),
 			PostRestoreVerify:    mysqlReady(false, false, CSIDataMover),
 		}, nil),
+		Entry("Mongo application Block Device", BackupRestoreCase{
```

Comment:
maybe "Mongo application Block Device DATAMOVER"

Reply:
I will change that once there are more block devices tests if that's ok for you. That's the first one to go, once it passes CI and merges more will be added.

Reply:
Changed also BlockDevice is one word now, so:

`Mongo application BlockDevice DATAMOVER`